two_sum.py

```diff
@@ -0,0 +1,17 @@
+from typing import List
+class Solution:
```

Comment:
```suggestion
from typing import List


class Solution:
```

So one thing you'll see a lot once you start working is comments like these that start with the word "nit" -- they're intended to be clear that the reviewer (me) is just being nitpicky about something. They don't expect you to take their suggestion, but they would like you to :) Basically it's a token of goodwill, while expressing their preference (as it turns out, there are A LOT of preferences and opinions in coding :P )

Nit: between imports and code, there should be two spaces

Reply:
Cool, I do not mind. I understand they are coming from a good place and I will these Nit's very helpful. Please keep them coming 👍🏽